src/js/providers/video-listener-mixin.js

```diff
@@ -56,7 +59,7 @@ const VideoListenerMixin = {
             return;
         }
 
-        if (!this.video.paused && (this.state === STATE_STALLED || this.state === STATE_LOADING)) {
```

Comment:
Provider state should never be set to buffering. Providers use `STATE_LOADING` and `STATE_STALLED` which both map to `STATE_BUFFERING` in the player.

Reply:
👍